x-pack/plugins/security_solution/public/overview/components/host_overview/endpoint_overview/index.tsx

```diff
@@ -20,79 +20,73 @@ import * as i18n from './translations';
 interface Props {
   contextID?: string;
   data: EndpointFields | null;
-  isInDetailsSidePanel?: boolean;
```

Comment:
These changes were just about removing `isInDetailsSidePanel`. It isn't necessary for these components to be aware of this